btree/btree.c

```diff
@@ -2705,17 +2777,16 @@ static struct m0_btree_oimpl *level_alloc(int height)
 	return oi;
 }
 
-static void level_cleanup(struct m0_btree_oimpl *oi,
+static void level_cleanup(struct m0_btree_oimpl *oi, struct td *tree,
```

Comment:
As I mentioned above the tree_put does not need a pointer to the tree descriptor so the tree pointer parameter to level_cleanup can also be removed.

Reply:
Thanks, removed tree pointer.